index.js

```diff
@@ -8,6 +8,7 @@ var easyButton = document.querySelector(".easy");
 var hardButton = document.querySelector(".hard");
 var newColors = document.querySelector(".new-game");
 var squaresList = document.querySelectorAll("li.square");
+var modeButtons = document.querySelectorAll(".mode");
```

Comment:
use `const` instead (instead of `var` which u actually should never use https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75)

Reply:
This object was removed